modules/vaos/app/controllers/vaos/messages_controller.rb

```diff
@@ -6,8 +6,22 @@ def index
       render json: MessagesSerializer.new(messages[:data], meta: messages[:meta])
     end
 
+    def create
+      response = messages_service.post_message(appointment_request_id, post_params)
+      render json: MessagesSerializer.new(response[:data], meta: response[:meta])
+    end
+
     private
 
+    def appointment_request_id
+      params[:appointment_request_id]
+    end
+
+    def post_params
+      params.require(:message_text)
```

Comment:
You could do:

`params.require(:message_text).permit(:message_text)`

Reply:
i know, but sometimes you have an array of .requires keys and thats where things get tripped up. I haven't seen a clearly defined convention for this across the code. I personally think it's a poor design choice by Rails maintainers to do this and they ought to favor this happening in form objects, but convention over configuration even at the expense of separation of concerns is the rails way!